src/client/datascience/commands/commandRegistry.ts

```diff
@@ -81,7 +81,7 @@ export class CommandRegistry implements IDisposable {
         this.registerCommand(Commands.RunCell, this.runCell);
         this.registerCommand(Commands.RunCurrentCell, this.runCurrentCell);
         this.registerCommand(Commands.RunCurrentCellAdvance, this.runCurrentCellAndAdvance);
```

Comment:
Okay for me to make further changes after the PR's been approved? If so I'm happy to remove these redundant `= undefined`s.

Reply:
Yeah definitely. You also have to fix the linter errors before we can submit. 

The other failures - the VS code test failures - are a result of API changes from VS code affecting our tests and aren't related to your PR (so we can submit with them failing). But the linter errors have to be fixed.

Thanks

Reply:
I tried removing the `= undefined`s but in both cases I get TypeScript errors:

![undefined](https://user-images.githubusercontent.com/23294156/111863702-868b9780-89a0-11eb-8382-db76fd6600f9.png)

Reply:
You have to fix the Commands.ts file so that the parameters for ExecSelectionInInteeractiveWindow match the value you're using.

Then the second error means you should change the (text: string | undefined) to (text?: string) because the argument is actually optional.

Reply:
See this spot here:
https://github.com/microsoft/vscode-jupyter/blob/ea611e5faf36c22abd2589085f03f81869365970/src/client/common/application/commands.ts#L35